linearRegression.ipynb

```diff
@@ -0,0 +1,108 @@
Pi4+P193//90E1Pp9PW7ZsCdT0pf749GRsZopcDnu7Her9bDo7e2xsZkpfNgsAgC4L+fT5QYMGKTs7O+jYwIEDNWTIkMDxGTNmaN68eUpJSVFSUpIefPBB5ebmaty4cZKkiRMnKisrS/fcc49KSkrkdrv1y1/+UrNmzVJCQoIk6Wc/+5leeOEF/eIXv9BPf/pTbd26VevWrdOmTZsC1503b56mT5+uMWPGaOzYsXruued08uRJ3XfffaG+7Yvqracn4ZyBFhtjU3FhlmauqZZNCur287eguDAramfEAQD6v7Bsuvqf//mfiomJ0Z133qlTp04pPz9fL730UuB8bGysNm7cqJkzZyo3N1cDBw7U9OnT9e///u+BmszMTG3atEkPPfSQnn/+eV111VX6r//6r8AaQpJ099136+uvv9aiRYvkdrs1cuRIlZeXtxtA3Rf8T0/cnpYOxwnZJDm7+fQkEtbvKch2qXTa6HbtcLKOEAAgCvTaOkL9QajXEfLPGpM6fnrSnTV3Im39nv60NhIAILpFxDpCaM//9MTpCO7+cjrs3QoukTgDLTbGptxhQ/TDkVcqd9gQQhAAICqEpWvMygqyXboty3lJT0+6MwMtd9iQELQaAID+iSAUBv6nJz3VH2egAQAQDnSNRSHW7wEAIDQIQlGI9XsAAAgNglAU8q/fI6ldGGL9HgAAuo4gFKVCNQMNAAArY7B0FAvFDDQAAKyMIBTlLnUGGgAAVkbXGAAAsCyCEAAAsCyCEAAAsCyCEAAAsCyCEAAAsCyCEAAAsCyCEAAAsCyCEAAAsCyCEAAAsCxWlr4AY4wkyev1hrklAACgq/y/2/7f8QshCF3A8ePHJUkZGRlhbgkAAOiu48ePy+FwXLDGZroSlyzK5/Ppq6++0qBBg2SzWXcjU6/Xq4yMDB06dEhJSUnhbo7l8P2HF99/ePH9h1e0fv/GGB0/flzp6emKibnwKCCeCF1ATEyMrrrqqnA3I2IkJSVF1f8R+hu+//Di+w8vvv/wisbv/2JPgvwYLA0AACyLIAQAACyLIISLSkhIUHFxsRISEsLdFEvi+w8vvv/w4vsPLyt8/wyWBgAAlsUTIQAAYFkEIQAAYFkEIQAAYFkEIQAAYFkEIXRq6dKl+t73vqdBgwYpNTVVkydPVn19fbibZUnLli2TzWbT3Llzw90US/nyyy81bdo0DRkyRImJiRoxYoQ+/PDDcDer32ttbdXChQuVmZmpxMREDRs2TL/61a+6tG8Uembbtm0qLCxUenq6bDabNmzYEHTeGKNFixbJ5XIpMTFReXl5OnjwYHgaG2IEIXTq/fff16xZs/TBBx9o8+bNOn36tCZOnKiTJ0+Gu2mWsmvXLr388sv67ne/G+6mWMrRo0d18803a8CAAXr33XdVV1enZ599VoMHDw530/q95cuXq7S0VC+88IL279+v5cuXq6SkRL/+9a/D3bR+6+TJk7rxxhv14osvdni+pKREK1as0MqVK1VZWamBAwcqPz9fLS0tfdzS0GP6PLrs66+/Vmpqqt5//33deuut4W6OJZw4cUKjR4/WSy+9pCeeeEIjR47Uc889F+5mWcKCBQu0Y8cO/fGPfwx3UyznjjvuUFpamn7zm98Ejt15551KTEzUmjVrwtgya7DZbHrzzTc1efJkSWefBqWnp+vhhx/WI488IknyeDxKS0vT6tWrNWXKlDC29tLxRAhd5vF4JEkpKSlhbol1zJo1S5MmTVJeXl64m2I5b7/9tsaMGaMf//jHSk1N1ahRo/TKK6+Eu1mWcNNNN2nLli365JNPJEkff/yxtm/frh/84Adhbpk1NTQ0yO12B/13yOFwKCcnRxUVFWFsWWiw6Sq6xOfzae7cubr55puVnZ0d7uZYwuuvv67q6mrt2rUr3E2xpD//+c8qLS3VvHnz9G//9m/atWuXfv7znys+Pl7Tp08Pd/P6tQULFsjr9Wr48OGKjY1Va2urnnzySU2dOjXcTbMkt9stSUpLSws6npaWFjgXzQhC6JJZs2aptrZW27dvD3dTLOHQoUOaM2eONm/eLLvdHu7mWJLP59OYMWP01FNPSZJGjRql2tparVy5kiDUy9atW6e1a9fqtdde0w033KCamhrNnTtX6enpfPcIObrGcFGzZ8/Wxo0b9d577+mqq64Kd3MsYffu3Tp8+LBGjx6tuLg4xcXF6f3339eKFSsUFxen1tbWcDex33O5XMrKygo6dv311+vzzz8PU4us49FHH9WCBQs0ZcoUjRgxQvfcc48eeughLV26NNxNsySn0ylJampqCjre1NQUOBfNCELolDFGs2fP1ptvvqmtW7cqMzMz3E2yjAkTJmjv3r2qqakJvMaMGaOpU6eqpqZGsbGx4W5iv3fzzTe3Wy7ik08+0dChQ8PUIuv45ptvFBMT/PMUGxsrn88XphZZW2ZmppxOp7Zs2RI45vV6VVlZqdzc3DC2LDToGkOnZs2apddee01vvfWWBg0aFOgLdjgcSkxMDHPr+rdBgwa1G4s1cOBADRkyhDFafeShhx7STTfdpKeeekp33XWXqqqqtGrVKq1atSrcTev3CgsL9eSTT+rqq6/WDTfcoI8++kj/8R//oZ/+9Kfhblq/deLECf3pT38KvG9oaFBNTY1SUlJ09dVXa+7cuXriiSd0zTXXKDMzUwsXLlR6enpgZllUM0AnJHX4+u///u9wN82S/uEf/sHMmTMn3M2wlLKyMpOdnW0SEhLM8OHDzapVq8LdJEvwer1mzpw55uqrrzZ2u9185zvfMY8//rg5depUuJvWb7333nsd/vd++vTpxhhjfD6fWbhwoUlLSzMJCQlmwoQJpr6+PryNDhHWEQIAAJbFGCEAAGBZBCEAAGBZBCEAAGBZBCEAAGBZBCEAAGBZBCEAAGBZBCEAAGBZBCEAAGBZBCEAAGBZBCEAAGBZBCEAAGBZBCEAAGBZ/w8KFvzdhWmbmQAAAABJRU5ErkJggg==",
```

Comment:
the plot looks good but is missing features. When you will be providing explanations to someone else it is important to show:
1. title of the graph
2. title of the axis plus the unit (currency, cnt, percent)